packages/StepTracker/StepTracker.modules.scss

```diff
@@ -0,0 +1,83 @@
+@import '~@tds/core-colours/colours';
+@import '~@tds/core-responsive/responsive';
+
+.container {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+}
+
+.step {
+  width: 100%;
+  position: relative;
+  text-align: center;
+
+  &::before,
+  &::after {
+    position: absolute;
+    top: 17px;
+    width: calc(50% - 1.07rem);
+    content: ' ';
+    border-bottom: 0.1rem solid $color-shuttle-grey;
+  }
+
+  &::before {
+    left: 0;
+  }
+
+  &::after {
+    right: 0;
+  }
+
+  .icon {
+    display: inline-block;
+    border: 0.1rem solid $color-shuttle-grey;
+    border-radius: 50%;
+    font-size: 1.2rem;
+    width: 35px;
+    height: 35px;
+    margin-bottom: 1rem;
+  }
+
+  .label {
+    position: relative;
+    width: 100%;
+    display: none;
+    text-align: center;
```

Comment:
I think you can achieve the same behaviour of centered text with flexbox and the Flexbox component.... then we won't need as many custom styles here.

```jsx
// will need to add the "center" value for `justifyContent`
<Flexbox direction="row" justifyContent="center">
```

Reply:
Flexbox only allows a `justifyContent` value of `spaceBetween` though, which doesn't seem like something we want.

Reply:
We are in full control over the Flexbox component! :) And it is NOT a public component so there is no hesitation to changing the interface. Is there any reason not to add a new accepted value to the `justifyContent` prop?